app/lib/reducers/ui.js

```diff
@@ -24,7 +24,15 @@ const initial = Immutable({
   fontFamily: 'Menlo, "DejaVu Sans Mono", "Lucida Console", monospace',
   fontSizeOverride: null,
   css: '',
-  termCSS: '',
+  termCSS: '' +
+    'x-row {' +
+    '  text-shadow: 0 0 0 #fff;' +
```

Comment:
this removes the default input cursor. not sure on how to make this respect the color set for the text @rauchg